lib/bolt/config.rb

```diff
@@ -1,12 +1,11 @@
-require 'logger'
+require 'bolt/logger'
 require 'yaml'
 require 'bolt/cli'
 
 module Bolt
   Config = Struct.new(
     :concurrency,
     :format,
-    :log_destination,
```

Comment:
You removed log_destination here but not log_level, and removed defaults for both later. Seems like log_level could also be removed.

Reply:
No the default is handled elsewhere but it's still configurable. We never allowed log_destination to be configured.

Reply:
We still configure loglevel we just manage the default in the logger. log_destination is not configured yet and never has been.